README.md

```diff
@@ -150,6 +150,8 @@ autovacuum_analyze_scale_factor = 0.025
 
 The following steps are intended for Sensu Engineering use, they are shared here for transparency.
 
+**Check the pins in the #engineering channel in Slack for the IP of the SSH jump host.**
```

Comment:
There is now a DNS name associated with the IP address, it is `spdc.sensu.io`.

For now, people will still need to use the IP address for accessing the graphana dashboard, but you can already use this DNS name for connecting to the SSH server.

Reply:
Good to know, I'll add that as well!